### PR TITLE
Add SBOM creation to vNext, Northstar and v8 builds

### PR DESCRIPTION
**Pull request by micahgodbolt**
This PR updates our builds to include the creation of a [Software Bill of Materials](https://www.cisa.gov/sbom) 

This is only needed for compliance purposes, as we don't use it internally.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |   `54.925 kB`<br />`17.57 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `43.056 kB`<br />`12.381 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `20.828 kB`<br />`6.533 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `21.696 kB`<br />`6.832 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |    `21.79 kB`<br />`6.512 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `27.942 kB`<br />`8.023 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `33.192 kB`<br />`8.983 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `29.596 kB`<br />`8.594 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `35.682 kB`<br />`9.666 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |   `37.219 kB`<br />`8.634 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `48.488 kB`<br />`14.347 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `43.997 kB`<br />`13.195 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `7.618 kB`<br />`3.232 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `8.885 kB`<br />`3.673 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `7.811 kB`<br />`3.361 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `165.697 kB`<br />`46.873 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.489 kB`<br />`10.636 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `15.269 kB`<br />`5.523 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `10.075 kB`<br />`3.942 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `21.474 kB`<br />`7.118 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `8.323 kB`<br />`3.479 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `11.072 kB`<br />`4.496 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `102.827 kB`<br />`31.706 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `105.078 kB`<br />`32.072 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `95.348 kB`<br />`29.017 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |    `6.249 kB`<br />`2.163 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                                                         |   `22.808 kB`<br />`7.935 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `13.972 kB`<br />`5.237 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `22.936 kB`<br />`7.751 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |    `6.787 kB`<br />`2.887 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `25.397 kB`<br />`8.208 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `10.769 kB`<br />`4.224 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `14.077 kB`<br />`4.565 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |   `29.426 kB`<br />`6.551 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |     `18.42 kB`<br />`5.27 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |    `42.47 kB`<br />`14.62 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                                       |          `189 B`<br />`161 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/240b1c16b5493d3d2497af761f9c179be992378f'>240b1c16b5493d3d2497af761f9c179be992378f</a></sub>
<div id="bundleSizeComment9423"/>

**Review by dannyvv — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Avatar-mount_base.html">984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Avatar-mount.html">972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/BaseButton-mount_base.html">1029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/BaseButton-mount.html">991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Breadcrumb-mount_base.html">2911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Breadcrumb-mount.html">2924</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ButtonNext-mount_base.html">569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ButtonNext-mount.html">571</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Checkbox-mount_base.html">1717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Checkbox-mount.html">1698</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/CheckboxBase-mount_base.html">1435</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/CheckboxBase-mount.html">1436</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ChoiceGroup-mount_base.html">5245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ChoiceGroup-mount.html">5224</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ComboBox-mount_base.html">1064</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ComboBox-mount.html">1085</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/CommandBar-mount_base.html">11404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/CommandBar-mount.html">11411</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ContextualMenu-mount_base.html">9400</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ContextualMenu-mount.html">9320</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DefaultButton-mount_base.html">1281</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DefaultButton-mount.html">1263</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DetailsRow-mount_base.html">4075</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DetailsRow-mount.html">4093</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DetailsRowFast-mount_base.html">4093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DetailsRowFast-mount.html">4173</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DetailsRowNoStyles-mount_base.html">3868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DetailsRowNoStyles-mount.html">3930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Dialog-mount_base.html">2823</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Dialog-mount.html">2869</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DocumentCardTitle-mount_base.html">205</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/DocumentCardTitle-mount.html">188</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Dropdown-mount_base.html">3489</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Dropdown-mount.html">3536</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderNext-mount_base.html">2030</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderNext-mount.html">2048</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderWithTheme-mount_base.html">180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderWithTheme-mount.html">184</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderWithTheme-virtual-rerender.html">127</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">219</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">218</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FocusTrapZone-mount_base.html">2029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FocusTrapZone-mount.html">1951</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FocusZone-mount_base.html">1937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/FocusZone-mount.html">2024</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/IconButton-mount_base.html">1922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/IconButton-mount.html">1914</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Label-mount_base.html">396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Label-mount.html">415</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Layer-mount_base.html">3241</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Layer-mount.html">3273</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Link-mount_base.html">558</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Link-mount.html">563</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/MakeStyles-mount_base.html">1873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/MakeStyles-mount.html">1847</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/MenuButton-mount_base.html">1616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/MenuButton-mount.html">1637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/MessageBar-mount_base.html">2211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/MessageBar-mount.html">2250</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Nav-mount_base.html">3604</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Nav-mount.html">3566</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/OverflowSet-mount_base.html">1213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/OverflowSet-mount.html">1195</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Panel-mount_base.html">2763</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Panel-mount.html">2748</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Persona-mount_base.html">1003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Persona-mount.html">949</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Pivot-mount_base.html">1599</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Pivot-mount.html">1595</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/PrimaryButton-mount_base.html">1470</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/PrimaryButton-mount.html">1459</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Rating-mount_base.html">8317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Rating-mount.html">8483</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SearchBox-mount_base.html">1473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SearchBox-mount.html">1443</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Shimmer-mount_base.html">2762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Shimmer-mount.html">2757</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Slider-mount_base.html">2142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Slider-mount.html">2192</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SpinButton-mount_base.html">5417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SpinButton-mount.html">5498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Spinner-mount_base.html">490</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Spinner-mount.html">507</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SplitButton-mount_base.html">3497</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SplitButton-mount.html">3412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Stack-mount_base.html">608</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Stack-mount.html">617</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/StackWithIntrinsicChildren-mount_base.html">2519</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/StackWithIntrinsicChildren-mount.html">2513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/StackWithTextChildren-mount_base.html">5725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/StackWithTextChildren-mount.html">5772</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SwatchColorPicker-mount_base.html">12392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/SwatchColorPicker-mount.html">12340</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/TagPicker-mount_base.html">2914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/TagPicker-mount.html">2829</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/TeachingBubble-mount_base.html">14188</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/TeachingBubble-mount.html">14255</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Text-mount_base.html">487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Text-mount.html">494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/TextField-mount_base.html">1574</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/TextField-mount.html">1594</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ThemeProvider-mount_base.html">1316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ThemeProvider-mount.html">1320</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ThemeProvider-virtual-rerender_base.html">688</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ThemeProvider-virtual-rerender.html">703</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2067</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2072</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Toggle-mount_base.html">921</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/Toggle-mount.html">901</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/buttonNative-mount_base.html">164</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21613/perf-test/buttonNative-mount.html">163</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

**Comment by micahgodbolt** [comment]
Added the update to v8 and stardust build steps

**Comment by ling1726** [comment]
is this a duplicate of #21640 ?

**Comment by micahgodbolt** [comment]
> is this a duplicate of #21640 ?

So it seems! I didn't realize @dannyvv  was updating this repo as well.

File: azure-pipelines.release-fluentui.yml [hunk 1]
```diff
@@ -98,7 +98,16 @@ jobs:
           ContainerName: nightly-builds
           BlobPrefix: $(date) # upload tarballs to folder named by today's date
           Gzip: true
-
+      - task: ManifestGeneratorTask@0
+        displayName: 📒 Generate Manifest
+        inputs:
+          BuildDropPath: $(System.DefaultWorkingDirectory)
+      # These publish task push direclty to npm so manually archive the sbom package
```
[the remark this answers is not in the document]
Comment (micahgodbolt) [reply azure-pipelines.release-fluentui.yml:105]: it was copied over from Danny's PRs. Basically, we publish fluent ui assets to npm (not locally) so we need to manually save the sbom information.

[the remark this answers is not in the document]
Comment (ling1726) [reply azure-pipelines.release-fluentui.yml:105]: Let's add your explanation then, I can understand that better :)

**Comment by ling1726** [comment]
> > is this a duplicate of #21640 ?
> 
> So it seems! I didn't realize @dannyvv was updating this repo as well.

https://github.com/microsoft/fluentui/pull/21640#issuecomment-1032936869

@dannyvv  @micahgodbolt  can you close one of the PRs and bring all relevant changes so we have a single PR for this ? thanks

**Comment by micahgodbolt** [comment]
closing down as #21640 is more complete

**Pull request closed without merging**